locales/en-US/messages.properties

```diff
@@ -32,11 +32,10 @@ donate_button_uppercase=DONATE
 credit_card=Credit / debit card
 submitting=Submitting…
 
-# privacy policy variants
-privacy_policy_var_a=We care about privacy and about helping you make informed choices. We want you to understand that your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards) or <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">Paypal</a>, and a record of your donation will be stored by Mozilla.
+# privacy policy
 privacy_policy_var_b=Your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards) or <a href="https://www.paypal.com/us/webapps/mpp/ua/privacy-full">Paypal</a>, and a record of your donation will be stored by Mozilla.
-privacy_policy_var_c=By making a donation, you acknowledge that your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards) or by <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">Paypal</a> in accordance with their privacy policies, and a record of your donation will be stored by <a href="https://www.mozilla.org/privacy/">Mozilla</a>.
 privacy_policy_var_b_monthly=Your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards) or <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">Paypal</a>, and a record of your donation will be stored by Mozilla. Monthly donations are charged each month on the same day that you donate today, and will continue until you cancel. To cancel, email us at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.
+privacy_policy_var_b_monthly_refund=We are able to provide refunds up to one year after a donation is processed.
```

Comment:
@TheoChevalier I was just reviewing staging- this string needs to be always visible, AFAIK.